src/design-system/components/user-avatar/UserAvatar.tsx

```diff
@@ -29,8 +29,8 @@ const UserAvatar = forwardRef(
       variant = 'avatar',
       src,
       size = 'sm',
-      firstName,
-      lastName,
+      firstName = '',
+      lastName = '',
```

Comment:
If you add default value to those props, make them optional in `BaseUserAvatarProps`

Reply:
fair enough. fixed